napps/kytos/of_topology/README.rst

```diff
@@ -37,239 +29,49 @@ If you are going to install kytos-napps from source code, all napps will be
-             "name":"s3-eth3",
-             "mac":"62:21:7c:fe:f2:3f",
-             "id":"00:00:00:00:00:00:00:03:3",
-             "speed":"10 Gbps"
-          },
-          {
-             "switch":"00:00:00:00:00:00:00:03",
-             "type":"interface",
-             "port_number":65534,
-             "name":"s3",
-             "mac":"42:f9:63:76:99:43",
-             "id":"00:00:00:00:00:00:00:03:65534",
-             "speed":""
-          }
-       ],
-       "links":[
-          {
-             "type":"interface",
-             "source":"00:00:00:00:00:00:00:02",
-             "target":"00:00:00:00:00:00:00:02:1"
-          },
-          {
-             "type":"interface",
-             "source":"00:00:00:00:00:00:00:02",
-             "target":"00:00:00:00:00:00:00:02:2"
-          },
-          {
-             "type":"interface",
-             "source":"00:00:00:00:00:00:00:02",
-             "target":"00:00:00:00:00:00:00:02:3"
-          },
-          {
-             "type":"link",
-             "source":"00:00:00:00:00:00:00:02:3",
-             "target":"00:00:00:00:00:00:00:01:1"
-          },
-          {
-             "type":"interface",
-             "source":"00:00:00:00:00:00:00:02",
-             "target":"00:00:00:00:00:00:00:02:65534"
-          },
-          {
-             "type":"interface",
-             "source":"00:00:00:00:00:00:00:01",
-             "target":"00:00:00:00:00:00:00:01:1"
-          },
-          {
-             "type":"link",
-             "source":"00:00:00:00:00:00:00:01:1",
-             "target":"00:00:00:00:00:00:00:02:3"
-          },
-          {
-             "type":"interface",
-             "source":"00:00:00:00:00:00:00:01",
-             "target":"00:00:00:00:00:00:00:01:2"
-          },
-          {
-             "type":"link",
-             "source":"00:00:00:00:00:00:00:01:2",
-             "target":"00:00:00:00:00:00:00:03:3"
-          },
-          {
-             "type":"interface",
-             "source":"00:00:00:00:00:00:00:01",
-             "target":"00:00:00:00:00:00:00:01:65534"
-          },
-          {
-             "type":"interface",
-             "source":"00:00:00:00:00:00:00:03",
-             "target":"00:00:00:00:00:00:00:03:1"
-          },
-          {
-             "type":"interface",
-             "source":"00:00:00:00:00:00:00:03",
-             "target":"00:00:00:00:00:00:00:03:2"
-          },
-          {
-             "type":"interface",
-             "source":"00:00:00:00:00:00:00:03",
-             "target":"00:00:00:00:00:00:00:03:3"
-          },
-          {
-             "type":"link",
-             "source":"00:00:00:00:00:00:00:03:3",
-             "target":"00:00:00:00:00:00:00:01:2"
-          },
-          {
-             "type":"interface",
-             "source":"00:00:00:00:00:00:00:03",
-             "target":"00:00:00:00:00:00:00:03:65534"
-          }
-       ]
-    }
+event = KytosEvent(name='kytos/of_topology.change.port_down',
```

Comment:
Please add this line:

`The event is an instance of KytosEvent, with the following data:`